contracts/stellar-interchain-token/src/tests/test.rs

```diff
@@ -91,9 +91,7 @@ fn transfer_ownership_succeeds() {
     let (token, _) = setup_token(&env);
 
     assert_auth!(token.owner(), token.transfer_ownership(&new_owner));
-    goldie::assert!(fmt_emitted_event_at_idx::<OwnershipTransferredEvent>(
-        &env, -2
-    ));
+    goldie::assert!(fmt_last_emitted_event::<OwnershipTransferredEvent>(&env));
```

Comment:
breaking event change, be careful when you need to "fix" golden files once a contract is stabilized